spring-xd-starters/spring-xd-module-parent/publish-maven.gradle

```diff
@@ -162,7 +163,7 @@ def customizePom(pom, gradleProject) {
 				dependency {
 					groupId 'org.apache.hadoop'
 					artifactId 'hadoop-common'
-					version '2.3.0'
+					version '2.6.0'
```

Comment:
Ok, got it (scrap my earlier comment). Both the boot dependency and this were out of sync with `dependencies.properties`.